src/app/layout.tsx

```diff
@@ -1,13 +1,14 @@
-import type { Metadata } from 'next';
-import { Inter } from 'next/font/google';
-import '../styles/globals.css';
-import { ClerkProvider } from '@clerk/nextjs';
+import type { Metadata } from "next";
```

Comment:
please change your linting rules for ' otherwise you'll always have all this linting changes cluttering the files

Reply:
Sure no prob, just a quick one did you update your environment with the key for the db2? you should be able to https://app.planetscale.com/bookshift/bookshift/initial-setup/console
![image](https://github.com/bookshift/bookshift/assets/46777782/99f752fb-ae7b-4e3b-aa25-ed008f21298d)

Reply:
annoyingly you can only use the console for the branched db